powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/task/CleaningTask.java

```diff
@@ -75,4 +78,14 @@ public void expireActivations() {
         activationServiceBehavior.expireActivations();
     }
 
+    @Scheduled(fixedRateString = "${powerauth.service.scheduled.job.temporaryKeyCleanup:5000}")
+    @SchedulerLock(
+            name = "expireTemporaryKeys",
+            lockAtLeastFor = "#{T(java.lang.Math).round(${powerauth.service.scheduled.job.temporaryKeyCleanup:5000} * 0.8)}")
+    public void expireTemporaryKeys() {
+        LockAssert.assertLocked();
+        logger.debug("expireTemporaryKeys");
```

Comment:
Let's update the debug message, so it is consistent with other messages in this class, e.g. `Calling scheduled expiration of ...`.